src/main/java/edu/greenblitz/tobyDetermined/subsystems/DriveTrain/IO/GyroIOs/RealGyro.java

```diff
@@ -0,0 +1,37 @@
+package edu.greenblitz.tobyDetermined.subsystems.DriveTrain.IO.GyroIOs;
+
+import edu.greenblitz.tobyDetermined.RobotMap;
+import edu.greenblitz.utils.Gyros.NavX;
+import edu.greenblitz.utils.Gyros.PigeonGyro;
+
+public class RealGyro implements GyroIO{
+
+    PigeonGyro gyro;
+    NavX navX;
```

Comment:
Why do we have both a navx and a pigeon? Looks to me like we should have 2 kinds of 'Real' gyros

Reply:
i wanted to do a filter between the 2 gyros, but that's a project for another time